src/main/kotlin/ro/hibyte/betting/exceptions/AppAdvice.kt

```diff
@@ -0,0 +1,54 @@
+package ro.hibyte.betting.exceptions
+
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.ControllerAdvice
+import org.springframework.web.bind.annotation.ExceptionHandler
+import org.springframework.web.bind.annotation.ResponseBody
+import ro.hibyte.betting.exceptions.types.*
+import java.time.LocalDateTime
+
+@ControllerAdvice
+class AppAdvice {
+
+    @ResponseBody
+    @ExceptionHandler(value = [
+        BetTemplateNotFoundException::class,
+        BetTypeNotFoundException::class,
+        EventTemplateNotFoundException::class
+    ])
+    fun entityNotFoundHandler(ex: RuntimeException): ResponseEntity<Any> {
+        val body: MutableMap<String, Any> = mutableMapOf()
```

Comment:
Please create a new data class that represents an error (with `message`, `status` and `time` fields) instead of using a map